pkg/scheduler/priorities.go

```diff
@@ -80,3 +81,47 @@ func LeastRequestedPriority(pod api.Pod, podLister PodLister, minionLister Minio
 	}
 	return list, nil
 }
+
+type NodeLabelPrioritizer struct {
+	label    string
+	presence bool
+}
+
+func NewNodeLabelPriority(label string, presence bool) PriorityFunction {
+	labelPrioritizer := &NodeLabelPrioritizer{
+		label:    label,
+		presence: presence,
+	}
+	return labelPrioritizer.CalculateNodeLabelPriority
+}
+
+// CalculateNodeLabelPriority checks whether a particular label exists on a minion or not, regardless of its value
+// Consider the cases where the minions are places in regions/zones/racks and these are identified by labels
+// In some cases, it is required that only minions that are part of ANY of the defined regions/zones/racks be selected
```

Comment:
This doesn't seem like a compelling example. For the use case you described, you would probably set labels like "region" or "zone" and then put an identifier for the region or zone as the corresponding value and then have a selector that is an OR of different acceptable region ro zone values. (I guess you were assuming you would have labels like "region1" "region2" etc., but that seems more confusing to me.) I'm not saying not to include this priority function, but I don't think you'd use it for what you're describing.